SPECS/git/git.spec

```diff
@@ -39,6 +39,61 @@ Requires: git >= 2.1.2
 %description lang
 These are the additional language files of git.
 
+
+%global with_daemon 1
```

Comment:
I think we usually declare globals at the top

Reply:
Sure, i though i would keep them close to where they are used .. but sure .. moving them to a single location.